### PR TITLE
Complete design of apple archive website

### PR DESCRIPTION
**Pull request by collinsugwu**
what is PR contains
1 nav bar floated accordingly
2 background img with text on it
3 list of products positioned with flex
4 footer

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+        <div class="main">
+            <section class="header">
+                <div class="background">
+                    <ul>
+                        <li class="nav"><a href="#">
+                                <ion-icon class=".ion-icon" name="logo-apple"></ion-icon>
+                            </a></li>
+                        <li class="nav"><a href="#">Store</a></li>
+                        <li class="nav"><a href="#">Mac</a></li>
+                        <li class="nav"><a href="#">iPod</a></li>
```
Comment (uche-inyama) [inline index.html:21]: Wrap your navigation links with a navigation tag **```<nav> tag```** Look at [semantic html](https://www.w3schools.com/html/html5_semantic_elements.asp)

File: index.html [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+        <title>Apple</title>
+        <link rel="stylesheet" type="text/css" href="vendor/css/normalize.css">
+        <link rel="stylesheet" type="text/css" href="resources/css/main.css">
+        <link href="https://fonts.googleapis.com/css?family=Lato:100,300,400&display=swap" rel="stylesheet">
+    </head>
+
+    <body>
+        <div class="main">
+            <section class="header">
+                <div class="background">
```
Comment (uche-inyama) [inline index.html:14]: Use main tag  **```<main> tag```** to wrap the main content of your html document, Look at [Sematic Html](https://www.w3schools.com/html/html5_semantic_elements.asp).

**Review by uche-inyama — COMMENTED** [review]
## Hi Collins, nice job,

Let's make few updates :slightly_smiling_face: 

- [ ] Push your [navigation design](https://raw.githack.com/collinsugwu/Apple-clone/feature/index.html) down alittle bit, compare with the [original site](https://web.archive.org/web/20140301004610/http://www.apple.com/).

- [ ] Add a search icon inside your search field.
 
- [ ]  Also, look at the comments I made below, in your index.html file, about using semantic html :arrow_down: 

[**Note**] You can only merge a pull request, when it is approved. Ensure you send a link for an open pull request, while submitting another form for code review :slightly_smiling_face: 

### Implement these changes, then submit another form for code review

**```Happy coding```**

File: index.html (hunk 1 above)
Comment (collinsugwu) [reply index.html:21]: I merged by mistake and I couldn't undo it.

**Pull request merged**